src/pages/ProductList/components/Products.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router-dom';
+import { ROUTES_URL } from '@/RootRouter';
+import ProductItem from './ProductItem';
+
+import type { ProductList } from '@/types';
+
+interface ProductItemProps {
+  products?: ProductList;
+}
+
+function Products({ products }: ProductItemProps) {
```

Comment:
```suggestion
function Products({ products = [] }: ProductItemProps) {
```
optional하게 받고 별도로 undefined일때 추가적인 UI 분기가 없다면 default 값을 넣어줘도 될 것 같아요.

(마이너) 컴포넌트 네이밍이 조금 어색하다고 느껴지는데 ProductCards는 어떨지 살포시 제안드려봅니다 👀

Reply:
고민했는데, ProductCards 좋은거 같습니다😀